packages/core/aqua-to-js/src/common.ts

```diff
@@ -120,16 +120,10 @@ export function typeToTs(t: NonArrowType | ArrowWithoutCallbacks): string {
         return [name, typeToTs(type)];
       });
 
-      const generic =
-        args.length === 0
-          ? "null"
-          : args
-              .map(([name]) => {
-                return `'${name}'`;
-              })
-              .join(" | ");
-
-      args.push(["callParams", `CallParams$$<${generic}>`]);
+      // JS-client argument
```

Comment:
Can you please elaborate on this code? why `unlabeledProduct` means something, what is `callParams`, why `ParticleContext$$` is pushed as a value and `callParams` as a key?

Reply:
`arrow type` which has domain tag equal to `unlabeledProduct` means callback function, passed in aqua function as an argument.

`particleContext` - additional argument which passed in every service registered function.

Here i tried to disable passing additional param in aqua function callback arguments but after reading your comment and some thinking i removed this condition. Because callback in aqua function arguments registered in special `callbackSrv` and this behavior would be consistent